fhir-profile/src/test/java/com/ibm/fhir/profile/test/BPConstraintGeneratorTest.java

```diff
@@ -21,34 +18,23 @@
 import com.ibm.fhir.registry.FHIRRegistry;
 
 public class BPConstraintGeneratorTest {
+    private static final boolean DEBUG = true;
```

Comment:
Shouldn't we try to reduce the amount of output in the build log unless there's an error?

Reply:
yeah, it used to always print.  i made it conditional.  fine with turning this to false now.
```suggestion
    private static final boolean DEBUG = false;
```

Reply:
done